src/jsx-parser.ts

```diff
@@ -35,20 +35,23 @@ function getQualifiedElementName(elementName: JSXNode.JSXElementName): string {
     let qualifiedName;
 
     switch (elementName.type) {
-        case JSXSyntax.JSXIdentifier:
+        case JSXSyntax.JSXIdentifier: {
```

Comment:
Is there also a rule/option somewhere that minimize this unnecessary change?

Reply:
No [there isn't](https://eslint.org/docs/rules/no-case-declarations). Do you want me to revert that one?

Reply:
Yes, please!